manifests/0000_51_olm_06_deployment.yaml

```diff
@@ -71,6 +71,8 @@ spec:
           value: quay.io/openshift/origin-olm-catalogd:latest
         - name: OPERATOR_CONTROLLER_IMAGE
           value: quay.io/openshift/origin-olm-operator-controller:latest
+        - name: RELEASE_VERSION
+          value: "0.0.1-snapshot"
```

Comment:
Oh, I see we already have `OPERATOR_IMAGE_VERSION` set to `0.0.1-snapshot`, which is the magic string that is replaced with the payload version.

I think we should use that.